.env.example

```
@@ -1,4 +1,4 @@
-APP_NAME=Goravel
+APP_NAME=SMEDI SME Database
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)